src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <section class="product" data-qa="card">
+      <img 
+        class="product__image image" 
+        src="./images/imac.jpeg" 
+        alt="imac"
+      >
+
+      <div class="product__discription">
```

Comment:
```suggestion
      <p class="product__description">
```

Reply:
why I can't  use wrapper DIV?

Reply:
you can but in the checklist, it requires to use of semantic tags 
`p.12 [SEMANTICS] - Use semantic tags like h2, p ...`